saleor/graphql/product/schema.py

```diff
@@ -123,6 +130,14 @@ def resolve_collection(self, info, id):
     def resolve_collections(self, info, query=None, **kwargs):
         return resolve_collections(info, query)
 
+    @permission_required(['product.manage_products',])
```

Comment:
If there is single permission, it can be passed as a string: `@permission_required('product.manage_products')`.